templates/base_index.html

```diff
@@ -521,7 +521,7 @@ <h2 class="p-heading--1 u-no-margin--bottom u-no-max-width">Multi-cloud Kubernet
               <a href="/kubernetes/managed">Fully managed Kubernetes</a> on-prem or public cloud
             </li>
             <li class="p-list__item is-ticked">
-              <a href="https://juju.is/docs/kubernetes">Model-driven Kubernetes Operators</a>
+              <a href="https://juju.is/docs">Model-driven Kubernetes Operators</a>
```

Comment:
I wonder if there is documentation on "Model-driven Kubernetes Operators" specifically. Cause https://juju.is/docs takes you to Juju documentation, which does not sound the same to me.

Reply:
I have already left a comment in the docs and reached out to the docs team. This can be addressed in a separate PR when we get a response